modules/project-docs/pages/third-party-integrations.adoc

```diff
@@ -0,0 +1,31 @@
+= Integrations & Ecosystem
+:navtitle: Integrations
+:page-topic-type: project-doc
+:page-aliases: 
+
+[abstract]
+
+The Couchbase Scala SDK is often used with unofficial and third party tools and applications to integrate into broader language and platform ecosystems, and across data lakes in heterogeneous environments.
+
+
+include::6.6@sdk:shared:partial$integrations.adoc[tag=intro]
+
+
+include::6.6@sdk:shared:partial$integrations.adoc[tag=official]
+
+The Couchbase Scala SDK is a first class citizen in the https://spring.io/projects/spring-data-couchbase[Spring Data^] world, and the Scala SDK can leverage that through the JVM.
+
+Couchbase also supports integrating with xref:2.4@spark-connector:ROOT:getting-started.adoc.adoc[Spark].
+
+
+include::6.6@sdk:shared:partial$integrations.adoc[tag=important]
+
+Why not take advantage of compatible Java tools? 
+Many dataflow tools integrate with Couchbase, including https://github.com/apache/nifi/tree/main/nifi-nar-bundles/nifi-couchbase-bundle[Apache NiFi], 
+https://wildfly-extras.github.io/wildfly-camel/#_camel_couchbase[Apache Camel],
+and https://github.com/couchbaselabs/flink-connector-couchbase[Apache Flink].
```

Comment:
missing external link markup `^`

Reply:
Got it - will merge this (older version, lower traffic) as is - and change in 1.2